test.py

```diff
@@ -29,17 +29,18 @@
 import traceback
```

Comment:
re: queue of tests:

It's an interesting idea.

The thing is once we have the context for running a suite (e.g. a `ScyllaCluster`) we should make the best by using it as much as possible (as Nadav pointed out before in other PRs).

Perhaps it could be factored in somehow. Will try later.